operations/local-replication/README.md

```diff
@@ -0,0 +1,499 @@
+
+# vault3 read supersecret/drtest
+## Key             	Value
+## ---             	-----
+## refresh_interval	768h0m0s
+## password        	baines
+## username        	harold
+
+## SUCCESS!
+
+```
+
+The environment looks like the following at this step:
+
+```
++---------------------------------+                    +------------------------------------+
+| vault port:8200                 |                    | vault2 port:8202                   |
+| Replication disabled            |                    | Performance secondary replication  |
+| (or demoted)                    |                    | vault3 --> vault2                  |
+|                                 |                    |                                    |
++---------------------------------+                    +------------------------------------+
+
+                                                                          ^
+                                                                          |
+                                                                          |
+                                                                          |
++---------------------------------+                                       |
+| vault3 port:8204                |                                       |
+| DR primary replication          |  +------------------------------------+
+| Performance primary replication |
+| vault3 --> vault2               |
++---------------------------------+
+```
+
+
+FAILBACK - Option 1  - relevant for Vault 0.8-0.9
+Note that this is not an idea situation today, as we must first sync DR replication set back to vault, then perform another failover such that vault is the perf primary/dr primary.
+```
+# Disable replication on vault (if not already done)
+vault write -f /sys/replication/dr/primary/disable
+vault write -f /sys/replication/performance/primary/disable
+
+# enable vault as DR secondary to vault3
+vault3 auth root
+vault3 write -f /sys/replication/dr/primary/enable
+PRIMARY_DR_TOKEN=$(vault3 write -format=json /sys/replication/dr/primary/secondary-token id=vault | jq --raw-output '.wrap_info .token' )
+sleep 10
+vault auth root
+vault write /sys/replication/dr/secondary/enable token=${PRIMARY_DR_TOKEN}
+sleep 10  
+```
+
+FAILBACK - Option 2 - relevant for Vault >= 0.9.1
+This scenario assumes the primary was demoted
+```
+# Promote original Vault instance back to disaster recovery primary
+vault write -f /sys/replication/dr/secondary/promote dr_operation_token=${DR_OPERATION_TOKEN}
+vault write -f /sys/replication/dr/primary/enable
+NEW_PRIMARY_DR_TOKEN=$(vault write -format=json /sys/replication/dr/primary/secondary-token id=vault3 | jq --raw-output '.wrap_info .token' )
+vault3 write -f /sys/replication/dr/primary/demote
+vault3 write /sys/replication/dr/secondary/update-primary primary_api_addr=127.0.0.1:8200 token=${NEW_PRIMARY_DR_TOKEN}
+
+# Promote original Vault instance back to performance primary
```

Comment:
I'm not sure this step is needed. If the previous DR primary was a perf primary then the promoted DR secondary should automatically become the perf primary